tests/test_create.py

```diff
@@ -220,6 +220,17 @@ def test_create_node_parent_retained_on_create(node: Any) -> None:
     cfg2 = OmegaConf.create({"zonk": cfg1.foo})
     assert cfg2 == {"zonk": node}
     assert cfg1.foo._get_parent() == cfg1
+    assert id(cfg1.foo._get_parent()) == id(cfg1)
```

Comment:
Better use `is` rather than `id`:
```suggestion
    assert cfg1.foo._get_parent() is cfg1
```
(same for other `id`-based comparisons below)

Reply:
Also I think a test is also needed to check that we don't make a copy anymore. Would the following assert pass?
```python
assert cfg1._get_node("foo") is node
```

Reply:
Ok => resolving